go.mod

```diff
@@ -18,6 +18,7 @@ replace (
 
 // Internal deps fix version
 replace (
+	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf
```

Comment:
does setting `v2.6` didn't works?

Reply:
Unfortunately not, most likely because it's not following the `vx.y.z`

Reply:
```suggestion
	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf // v2.6
```